<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Denis-PC" ignored="8" name="TrigonometryTest" tests="18" failures="9" timestamp="04 июн 2017 13:27:07 GMT" time="0.076" errors="0">
  <testcase name="testCos" time="0.003" classname="testng.calculator.CosTest">
    <failure type="java.lang.AssertionError" message="expected [1.0] but found [0.0]">
      <![CDATA[java.lang.AssertionError: expected [1.0] but found [0.0]
	at testng.calculator.CosTest.testCos(CosTest.java:15)
	at testng.runner.TestRunner.main(TestRunner.java:33)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testCos -->
  <testcase name="testCos" time="0.0" classname="testng.calculator.CosTest">
    <failure type="java.lang.AssertionError" message="expected [0.866] but found [0.5]">
      <![CDATA[java.lang.AssertionError: expected [0.866] but found [0.5]
	at testng.calculator.CosTest.testCos(CosTest.java:15)
	at testng.runner.TestRunner.main(TestRunner.java:33)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testCos -->
  <testcase name="testCos" time="0.0" classname="testng.calculator.CosTest"/>
  <testcase name="testCos" time="0.0" classname="testng.calculator.CosTest">
    <failure type="java.lang.AssertionError" message="expected [0.5] but found [0.866]">
      <![CDATA[java.lang.AssertionError: expected [0.5] but found [0.866]
	at testng.calculator.CosTest.testCos(CosTest.java:15)
	at testng.runner.TestRunner.main(TestRunner.java:33)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testCos -->
  <testcase name="testCos" time="0.0" classname="testng.calculator.CosTest">
    <failure type="java.lang.AssertionError" message="expected [0.0] but found [1.0]">
      <![CDATA[java.lang.AssertionError: expected [0.0] but found [1.0]
	at testng.calculator.CosTest.testCos(CosTest.java:15)
	at testng.runner.TestRunner.main(TestRunner.java:33)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testCos -->
  <testcase name="testSin" time="0.001" classname="testng.calculator.SinTest"/>
  <testcase name="testSin" time="0.0" classname="testng.calculator.SinTest"/>
  <testcase name="testSin" time="0.0" classname="testng.calculator.SinTest"/>
  <testcase name="testSin" time="0.001" classname="testng.calculator.SinTest"/>
  <testcase name="testSin" time="0.0" classname="testng.calculator.SinTest"/>
  <testcase name="testTg" time="0.0" classname="testng.calculator.TgTest">
    <failure type="java.lang.AssertionError" message="expected [0.577] but found [1.0]">
      <![CDATA[java.lang.AssertionError: expected [0.577] but found [1.0]
	at testng.calculator.TgTest.testTg(TgTest.java:14)
	at testng.runner.TestRunner.main(TestRunner.java:33)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testTg -->
  <testcase name="testTg" time="0.0" classname="testng.calculator.TgTest"/>
  <testcase name="testTg" time="0.0" classname="testng.calculator.TgTest"/>
  <testcase name="testTg" time="0.0" classname="testng.calculator.TgTest">
    <failure type="java.lang.AssertionError" message="expected [1.732] but found [1.0]">
      <![CDATA[java.lang.AssertionError: expected [1.732] but found [1.0]
	at testng.calculator.TgTest.testTg(TgTest.java:14)
	at testng.runner.TestRunner.main(TestRunner.java:33)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testTg -->
  <testcase name="testCtg" time="0.0" classname="testng.calculator.CtgTest">
    <failure type="java.lang.AssertionError" message="expected [1.732] but found [1.0]">
      <![CDATA[java.lang.AssertionError: expected [1.732] but found [1.0]
	at testng.calculator.CtgTest.testCtg(CtgTest.java:14)
	at testng.runner.TestRunner.main(TestRunner.java:33)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testCtg -->
  <testcase name="testCtg" time="0.0" classname="testng.calculator.CtgTest"/>
  <testcase name="testCtg" time="0.0" classname="testng.calculator.CtgTest">
    <failure type="java.lang.AssertionError" message="expected [0.577] but found [1.0]">
      <![CDATA[java.lang.AssertionError: expected [0.577] but found [1.0]
	at testng.calculator.CtgTest.testCtg(CtgTest.java:14)
	at testng.runner.TestRunner.main(TestRunner.java:33)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testCtg -->
  <testcase name="testCtg" time="0.001" classname="testng.calculator.CtgTest">
    <failure type="java.lang.AssertionError" message="expected [0.0] but found [1.0]">
      <![CDATA[java.lang.AssertionError: expected [0.0] but found [1.0]
	at testng.calculator.CtgTest.testCtg(CtgTest.java:14)
	at testng.runner.TestRunner.main(TestRunner.java:33)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testCtg -->
  <testcase name="ctg0" time="0.0" classname="testng.calculator.CtgTest">
    <ignored/>
  </testcase> <!-- ctg0 -->
  <testcase name="tg90" time="0.0" classname="testng.calculator.TgTest">
    <ignored/>
  </testcase> <!-- tg90 -->
  <testcase name="@BeforeSuite setPhrase" time="0.0" classname="testng.calculator.BaseTest">
    <ignored/>
  </testcase> <!-- @BeforeSuite setPhrase -->
  <testcase name="@BeforeSuite setPhrase" time="0.0" classname="testng.calculator.BaseTest">
    <ignored/>
  </testcase> <!-- @BeforeSuite setPhrase -->
  <testcase name="@BeforeSuite setPhrase" time="0.0" classname="testng.calculator.BaseTest">
    <ignored/>
  </testcase> <!-- @BeforeSuite setPhrase -->
  <testcase name="@AfterSuite setPhraseDone" time="0.0" classname="testng.calculator.BaseTest">
    <ignored/>
  </testcase> <!-- @AfterSuite setPhraseDone -->
  <testcase name="@AfterSuite setPhraseDone" time="0.0" classname="testng.calculator.BaseTest">
    <ignored/>
  </testcase> <!-- @AfterSuite setPhraseDone -->
  <testcase name="@AfterSuite setPhraseDone" time="0.0" classname="testng.calculator.BaseTest">
    <ignored/>
  </testcase> <!-- @AfterSuite setPhraseDone -->
</testsuite> <!-- TrigonometryTest -->
